files/build_templates/docker_image_ctl.j2

```diff
@@ -327,6 +327,11 @@ function postStartAction()
             docker cp $PSENSOR pmon:/usr/bin/
         fi
     fi
+{%- elif docker_container_name == "eventd" %}
+    export EVENTD_STATE=$(sonic-db-cli -s CONFIG_DB HGET 'FEATURE|eventd' 'state')
+    j2 -f json --import-env=ENVIRONMENT /usr/share/sonic/templates/rsyslog_plugin/rsyslog_plugin.conf.j2 /usr/share/sonic/templates/rsyslog_plugin/events_info.json  > /etc/rsyslog.d/host_events.conf
+    j2 -f json --import-env=ENVIRONMENT /usr/share/sonic/templates/rsyslog_plugin/rsyslog_plugin.conf.j2 /usr/share/sonic/templates/rsyslog_plugin/syncd_events_info.json  > /etc/rsyslog.d/syncd_events.conf
+    systemctl restart rsyslog
```

Comment:
@zbud-msft @qiluo-msft In our testing this introduces a race between eventd's postStartAction and rsyslog-config service. rsyslog-config.service will overwrite /etc/rsyslog.conf and when at the same time we restart rsyslogd we'll get rsyslogd exited and then eventd service start failure.